SECURITY_RELEASE_PROCESS.md

```diff
@@ -0,0 +1,260 @@
+# Security Release Process
+
+Envoy is a large growing community of volunteers, users, and vendors. The Envoy community has
+adopted this security disclosures and response policy to ensure we responsibly handle critical
+issues.
+
+## Product Security Team (PST)
+
+Security vulnerabilities should be handled quickly and sometimes privately. The primary goal of this
+process is to reduce the total time users are vulnerable to publicly known exploits.
+
+The Product Security Team (PST) is responsible for organizing the entire response including internal
+communication and external disclosure but will need help from relevant developers to successfully
+run this process.
+
+The initial Product Security Team will consist of all [maintainers](OWNERS.md) in the private
+[envoy-security](https://groups.google.com/forum/#!forum/envoy-security) list. In the future we may
+decide to have a subset of maintainers work on security response given that this process is time
+consuming.
+
+## Disclosures
+
+### Private Disclosure Processes
+
+The Envoy community asks that all suspected vulnerabilities be privately and responsibly disclosed
+via the [reporting policy](README.md#reporting-security-vulnerabilities).
+
+### Public Disclosure Processes
+
+If you know of a publicly disclosed security vulnerability please IMMEDIATELY email
+[envoy-security](https://groups.google.com/forum/#!forum/envoy-security) to inform the Product
+Security Team (PST) about the vulnerability so they may start the patch, release, and communication
+process.
+
+If possible the PST will ask the person making the public report if the issue can be handled via a
```

Comment:
Drop "public" here, it's not public yet.

Reply:
It is public. This is the "public disclosure" variant where someone has posted something without telling us.